api-models/thrift/src/main/thrift/appsRendering.thrift

```diff
@@ -54,4 +67,5 @@ struct RenderingRequest {
     4: optional map<string,string> targetingParams
     5: optional Branding branding
     6: optional list<Campaign> campaigns
+    7: optional list<RelatedContent> relatedContent
```

Comment:
Would this mean that there are two situations in which we wouldn't want to render the related content component? One when this field is missing/`undefined`, and one when the list is empty?